metadata-service/auth-impl/src/main/java/com/datahub/authorization/DataHubAuthorizer.java

```diff
@@ -151,16 +152,17 @@ public AuthorizedActors authorizedActors(
       // Step 3: For each matching policy, add actors that are authorized.
       authorizedUsers.addAll(matchingActors.getUsers());
       authorizedGroups.addAll(matchingActors.getGroups());
-      if (matchingActors.allUsers()) {
+      authorizedRoles.addAll(matchingActors.getRoles());
+      if (matchingActors.getAllUsers()) {
         allUsers = true;
       }
-      if (matchingActors.allGroups()) {
+      if (matchingActors.getAllGroups()) {
         allGroups = true;
       }
     }
 
     // Step 4: Return all authorized users and groups.
-    return new AuthorizedActors(privilege, authorizedUsers, authorizedGroups, allUsers, allGroups);
+    return new AuthorizedActors(privilege, authorizedUsers, authorizedGroups, authorizedRoles, allUsers, allGroups);
```

Comment:
Same with this, the way role matches happen is through the PolicyEngine, not this method

Reply:
This one is actually used for action requests feature, to determine who should be placed as a reviewer

Reply:
Got it, must be new stuff I haven't pulled down yet then